packages/web-react/src/components/Toast/README.md

```diff
@@ -185,21 +187,51 @@ Alternatively, a custom icon can be used:
 | `success`     | `check-plain` |
 | `warning`     | `warning`     |
 
-### Action Link
+### ToastBar Components
+
+The content of `ToastBar` can be assembled from the following subcomponents:
+
+#### ToastBarMessage
 
-An action link can be added to the ToastBar component:
+`ToastBarMessage` is a subcomponent designated for the main message in `ToastBar`.
+
+Usage example:
 
 ```jsx
-<ToastBar id="my-toast">
-  Message with action
-  <Link href="#" color="inverted" isUnderlined>
-    Action
-  </Link>
+<ToastBar id="my-toast" isOpen={isOpen} onClose={() => setIsOpen(false)} isDismissible>
+  <ToastBarMessage>This is the main toast message.</ToastBarMessage>
 </ToastBar>
 ```
 
-👉 For the sake of flexibility, developers can pass the link as part of the message. However, it is strongly recommended
-to use the **inverted underlined** variant of the link (for all ToastBar colors) to make it stand out from the message.
+#### API
+
+| Name       | Type        | Default | Required | Description                    |
+| ---------- | ----------- | ------- | -------- | ------------------------------ |
+| `children` | `ReactNode` | —       | ✓        | Content of the ToastBarMessage |
```

Comment:
Please copypaste the section about escape hatches below. You can take it anywhere else.

Reply:
This should be everywhere (in twig and react)

Reply:
I'm not sure if I understand correctly. I added this part to web-twig README.

Reply:
This was meant in @crishpeen comment:
```
On top of the API options, the components accept [additional attributes][readme-additional-attributes].
If you need more control over the styling of a component, you can use [style props][readme-style-props]
and [escape hatches][readme-escape-hatches].
```
This paragraph says that the component is using `UNSAFE_` style props. E.g. we do not want to repeat them in every API table.